manim/utils/tex_templates.py

```diff
@@ -549,7 +549,7 @@ class TexTemplateLibrary(object):
 """
 )
 gnufsfs.tex_compiler = "xelatex"
-gnufsfs.output_format = ".pdf"
+gnufsfs.output_format = ".xdv"
```

Comment:
Cool, docs are not failing anymore!
But has the .xdv any advantages/disadvanteges over .pdf?
I mean, this would then also affect all offline rendering of tex files from tex_templates.
and probably for a user, it would be more intuitive to see a .pdf file in a folder for rendered tex then .xdv 
But as long as both works, I don't mind.

Reply:
No disadvantages as far as I can tell. @cobordism might have more to say about whether this change is okay or not.

Reply:
generally I don't have a problem with changing it to xdv.
Usually the reason something is set to pdf is because pdf worked on my machine when I wrote this, but xdv did not.

Make sure you have a new enough version of xelatex and dvisvgm installed. I don't know what was giving you the "unknown option --pdf" error, but that should never happen.

Reply:
latex produces .dvi, xelatex produces .xdv. These get converted to svg by dvisvgm.

If that doesn't work, we use can also compile to .pdf and use dvisvgm --pdf to convert to svg... but in general the former is preferable.

Reply:
The version of `dvisvgm` installed within the RTD containers is too old for the `--pdf` option, and we cannot upgrade that.